src/services/dto/courtCaseDto.ts

```diff
@@ -13,6 +18,10 @@ export const courtCaseToDisplayPartialCourtCaseDto = (courtCase: CourtCase): Dis
     errorLockedByUsername: courtCase.errorLockedByUsername,
     errorReport: courtCase.errorReport,
     errorStatus: courtCase.errorStatus ?? null,
+    canUserEditExceptions:
```

Comment:
I wonder if `canUserEditExceptions` could be defined elsewhere, it feels strange to pass current users as a dependency for court case DTO

Reply:
You've got everything you need before you get to the UI (still on the server). So it's encapsulated the DTO and you don't need to expensive lookups.

Reply:
So I think that's the right place for it be.